VCFv4.3.tex

```diff
@@ -503,41 +503,100 @@ \subsection{VCF tag naming conventions}
 
 \section{INFO keys used for structural variants}
 \label{sv-info-keys}
-The following INFO keys are reserved for encoding structural variants. In general, when these keys are used by imprecise variants, the values should be best estimates. When a key reflects a property of a single alt allele (e.g. SVLEN), then when there are multiple alt alleles there will be multiple values for the key corresponding to each alelle (e.g. SVLEN=-100,-110 for a deletion with two distinct alt alleles).
+The following INFO keys are reserved for encoding structural variants. When a key reflects a property of a single alt allele (e.g. SVLEN), then when there are multiple alt alleles there will be multiple values for the key corresponding to each alelle (e.g. SVLEN=-100,-110 for a deletion with two distinct alt alleles).
 \footnotesize
 \begin{verbatim}
 ##INFO=<ID=IMPRECISE,Number=0,Type=Flag,Description="Imprecise structural variation">
 ##INFO=<ID=NOVEL,Number=0,Type=Flag,Description="Indicates a novel structural variation">
 ##INFO=<ID=END,Number=1,Type=Integer,Description="End position of the variant described in this record">
 \end{verbatim}
 \normalsize
-For precise variants, END is POS + length of REF allele - 1, and the for imprecise variants the corresponding best estimate.
+For precise variants, END is POS + length of REF allele - 1.
 
 \footnotesize
 \begin{verbatim}
 ##INFO=<ID=SVTYPE,Number=1,Type=String,Description="Type of structural variant">
 \end{verbatim}
 \normalsize
-This key can be derived from the REF/ALT fields but is useful for filtering. The reserved values must be used for the types listed below:
+SVTYPE is required for all structural variants. The following reserved values should be used whenever possible; indented values are subtypes that should be used if evidence supports them (see examples in Section 5.3, Encoding Structural Variants):\vspace{0.3cm}\\
 \begin{itemize}
-  \item DEL: Deletion relative to the reference
-  \item INS: Insertion of novel sequence relative to the reference
-  \item DUP: Region of elevated copy number relative to the reference
-  \item INV: Inversion of reference sequence
-  \item CNV: Copy number variable region (may be both deletion and duplication)
+\item [\null] INS:  Insertion relative to the reference
+	\begin{itemize}
+	\item [\null] MEI:  Mobile element insertion relative to the reference
+		\begin{itemize}
```

Comment:
How is this list to be maintained? Do we have any ME ontologies we can reference so our list is more exhaustive?

Reply:
For this PR I included mobile element types from data published by 1000 Genomes: Alu, LINE1, SVA, and HERV. I added 'RET' (for 'retrotransposon') because I am aware of a specific need to accommodate it.

For now I think it is sufficient to use the types in this list; additional types can be added, and a formal ontology consulted, if and when the need to do so becomes more obvious.